docs/.vuepress/theme/components/AposPageMeta.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div v-if="editOrIssueLinks" class="meta">
+    <EditOrIssue />
+    <p class="meta-updated">
+      Last updated: {{$page.lastUpdated}}
+    </p>
+  </div>
+</template>
+
+<script>
+import EditOrIssue from './EditOrIssue.vue'
+
+export default {
+  name: 'AposPageMeta',
+  components: {
+    EditOrIssue
+  },
+  props: {
+    editOrIssueLinks: {
+      type: Boolean,
+      default: true
+    }
+  }
+}
+</script>
+
+<style lang="stylus" scoped>
+.meta {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  padding: 2rem 0 1rem;
+  font-size: 90%;
```

Comment:
Why 90%? I get nervous with percents in this case. I like these guys approach (https://engageinteractive.co.uk/blog/em-vs-rem-vs-px).

Reply:
I just wanted it `smaller`. But I can use a rem value instead.